components/form/InputWithSelect.vue

```diff
@@ -146,7 +153,9 @@ export default {
       :disabled="disabled"
       :required="textRequired"
       :mode="mode"
+      :validators="validators"
       v-bind="$attrs"
+      @setValid="(isValid) => { $emit('setValid', isValid) }"
```

Comment:
These could be simplified to just `@setValid="isValid => $emit('setValid', isValid)"`